testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -93,7 +105,7 @@ def test_long_traj(u):
 
 
 def test_not_universe_error(u):
-    trj_only = u.trajectory
+    trj_only = u.universe.trajectory
```

Comment:
can you briefly explain what this change is for? The `u` fixture returns a universe and now we need another `.universe` call?

Reply:
yeah, since `u` can also be an `AtomGroup`.  I should've also updated `match` as `match='U is not a Universe or AtomGroup'`. I will do that.

Reply:
@tylerjereddy is correct here, u.trajectory == u.universe.trajectory, there's no need to make this change, either way you are testing what happens if you pass a trajectory through (neither a universe or an atomgroup)